### PR TITLE
NAS-107393 / 20.10 / Nas 107393

### PR DESCRIPTION
**Pull request by dkmullen**
New dashboard for support and sys general components

### CODE REVIEW

**Comment by erinix** [comment]
screenshot-20200910-133255

**Comment by erinix** [comment]
screenshot-20200910-121633

**Comment by erinix** [comment]
screenshot-20200910-133326

**Comment by kmoore134** [comment]
@dkmullen On one of the screenshots, pretty sure we were going to avoid the "Submit" wording on buttons and use "Save" instead?

**Comment by erinix** [comment]
screenshot-20200910-133713

File: src/app/pages/system/general-settings/support/support.component.ts [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      case "FREENAS-MINI-3.0-E+":
+        this.product_image = 'freenas_mini_cropped.png';
+      break;
+      case "FREENAS-MINI-3.0-X":
+      case "FREENAS-MINI-3.0-X+":
+        this.product_image = 'freenas_mini_x_cropped.png';
+      break;
+      case "FREENAS-MINI-XL":
+      case "FREENAS-MINI-3.0-XL+":
+        this.product_image = 'freenas_mini_xl_cropped.png';
```
Comment (Mrt134) [inline src/app/pages/system/general-settings/support/support.component.ts:135]: Does this need to be updated to TrueNAS Mini or be able to handle either naming?

**Review by Mrt134 — APPROVED** [review]
Looking good! Had a question about FreeNAS/TrueNAS Mini naming, but everything else looks fine.

**Comment by william-gr** [comment]
Shouldnt Min and Max Pool match the width of the 1 column and the end of the Max Poll select align with the Address select?

**Comment by dkmullen** [comment]
Here's an updated screenshot of the NTP form with the min/max poll fields aligned and 'Save' as the action

Screenshot from 2020-09-11 09-06-07

File: src/app/pages/system/general-settings/support/support.component.ts (hunk 1 above)
Comment (dkmullen) [reply src/app/pages/system/general-settings/support/support.component.ts:135]: Will the new model names/numbers be reported  by middleware (system.info) the same as above, except with TRUENAS in place of FREENAS?

**Comment by william-gr** [comment]
@dkmullen  Sorry, I feel like I wasnt clear. I think its better to have Mix and Max Poll in two columns, but the very end of the Max Poll input box was not aligned to the very end of the address input box.

File: src/app/pages/system/general-settings/support/support.component.ts (hunk 1 above)
Comment (william-gr) [reply src/app/pages/system/general-settings/support/support.component.ts:135]: We need both for 3.0-*
FREENAS and TRUENAS

Comment (william-gr) [reply src/app/pages/system/general-settings/support/support.component.ts:135]: i think this changed was made in 12.0 and perhaps missed a backport to master?

**Comment by dkmullen** [comment]
@william-gr Sorry about that :) Here's an update

Screenshot from 2020-09-11 10-18-05

**Review by kmoore134 — APPROVED** [review]

**Comment by erinix** [comment]
This PR is superseded by https://github.com/freenas/webui/pull/4657, all of the commits are in there

**Pull request closed without merging**